_drafts/coop-values.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Coop Community Values"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags:
+  - community values
+---
+
+Having gone through a few [transitions]() in recent months along with global and local conversations surrounding the COVID-19 pandemic and rampant anti-blackness in the United States have caused those of us at the Coop to take a step back and really consider why we do what we do. From these discussions we have seen a need for and are beginning to develop the Coop's Community Values. 
```

Comment:
link? I'd also reword this paragraph to be less passive: "The Coop team would like to announce a new project for our community: development of the Coop's Community Values. Both recent transitions for the Coop and conversations surrounding...." (or something like that).